wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/episode/EpisodeViewModel.kt

```diff
@@ -103,6 +104,8 @@ class EpisodeViewModel @Inject constructor(
     )
 
     val stateFlow: StateFlow<State>
+    var showNowPlaying = MutableSharedFlow<Boolean>()
+        private set
```

Comment:
Do you think it might be worth adding a comment here noting that we're using a shared flow here because we need the event to only be sent one time and then discarded? Just to make it clear why we're not using a state flow like we normally do.

Also, the `private set var` on the mutable flow reads a bit odd to me because I read it as implying that we might want to switch out this flow for an entirely different flow. Maybe something like this would work:

``` kotlin
val _showNowPlaying = MutableSharedFlow<Boolean>()
val showNowPlaying = _showNowPlaying.asSharedFlow()
```

Reply:
Makes sense 👍 
Addressed in bd5a809